src/reducers/url-state.js

```diff
@@ -105,6 +109,19 @@ const selectedThread: Reducer<ThreadIndex | null> = (state = null, action) => {
     case 'ISOLATE_LOCAL_TRACK':
       // Only switch to non-null selected threads.
       return (action.selectedThreadIndex: ThreadIndex);
+    case 'HIDDEN_GLOBAL_TRACKS_REMOVED': {
+      if (state === null) {
+        return null;
+      }
+
+      let reduceBy = 0;
```

Comment:
As stated elsewhere, I don't think this approach is correct, as it needs a `Map<OldIndex, NewIndex>`.